python/README_Yaml_Template.md

```diff
@@ -14,10 +14,11 @@ on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplat
 
 ### Required Parameters
 
-* **yaml** (Input YAML file in Cloud Storage.): The input YAML file Dataflow reads from.
 
 ### Optional Parameters
 
+* **pipeline_spec** (Input YAML spec.): A yaml description of the pipeline to run.
```

Comment:
We should consider the names "yaml_pipeline" and "yaml_pipeline_file" or similar. (We can make aliases in main.py to keep things backwards compatible there if needed.)

Reply:
I agree those names are more friendly than using "spec" - That change will need to occur on main.py before I can change the template though. So if that name change and alias can't get cherry-picked into 2.53, I'll have to update this template after 2.54 is released.

Reply:
Not sure why I thought this^, must have been the holiday brain. I updated the template to use those names and created https://github.com/apache/beam/pull/29916 to add the aliases to main.py which, when merged, will allow this template being to be updated to remove the alias logic.